metpy/calc/kinematics.py

```diff
@@ -604,7 +604,7 @@ def absolute_vorticity(u, v, dx, dy, lats, dim_order='yx'):
         The grid spacing(s) in the y-direction. If an array, there should be one item less than
         the size of `u` along the applicable axis.
     lats : (M, N) ndarray
-        latitudes of the wind data
+        latitudes of the wind data in radians or appropriately united
```

Comment:
"in radians or with appropriate unit information attached"? We've used the attached idea elsewhere. I'm not sure of the proper spelling of "unit-ed"..."united"? The latter is spelled like the airline. I'm not sure if we're talking homographs or not.

Reply:
Yeah, that is exactly what I was thinking about when I was writing the initial draft and didn't like it too much either. I've implemented your version, which I think is still relatively succinct.